tests/common/test_statistic_points.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2024 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+
+from nncf.common.graph.transformations.commands import TargetPoint
+from nncf.common.graph.transformations.commands import TargetType
+from nncf.common.tensor_statistics.collectors import TensorStatisticCollectorBase
+from nncf.common.tensor_statistics.statistic_point import StatisticPoint
+from nncf.common.tensor_statistics.statistic_point import StatisticPointsContainer
+
+
+class TestStatisticPointsContainer:
+    @pytest.fixture
+    def container_with_statistics(self):
+        container = StatisticPointsContainer()
+        target_type = TargetType.LAYER
+        target_point = TargetPoint(target_type)
+        target_point.target_node_name = "Node"
+        # tensor_collector = TensorStatisticCollectorBase()
```

Comment:
Is it redundat comments?

Reply:
Yes, I forgot to remove it :/ Will commit in a sec